SWLOR.Game.Server/Perk/ForceAlter/DrainLife.cs

```diff
@@ -78,19 +78,19 @@ public void OnConcentrationTick(NWCreature creature, NWObject target, int spellT
             switch (spellTier)
             {
                 case 1:
-                    amount = 5;
+                    amount = 5 + ((int)ForceBalanceType.Dark - (int)ForceBalanceType.Light);
```

Comment:
This isn't doing what you think it is. You're subtracting two constants (Dark = 2, Light = 1) so this is always going to be (5 + (2 - 1)).

I assume you want to use the player's balance. It's completely slipping my mind how that's calculated and where it's stored but I would check the force resistance code for a place to start.

Reply:
Going to revert this for now, I will work on it but I can't get it how I want it then...